frontend/bower.json

```diff
@@ -2,22 +2,11 @@
   "name": "ng-boilerplate",
   "version": "0.3.1",
   "devDependencies": {
-    "angular-ui-tree": "JimLiu/angular-ui-tree#63a87768259750613b6a6f55a338dfb6b9a6fb8d",
-    "sassy-buttons": "",
-    "ng-token-auth": "https://git.lrdesign.com/lrd/ng-token-auth.git",
-    "angular-froala": "~1.2.0"
+    "sassy-buttons": ""
```

Comment:
at this point should we just copy the sassy buttons js file in to xing framework sass so we can get rid of bower entirely?